crates/sui-framework/sources/governance/validator_set.move

```diff
@@ -721,6 +723,7 @@ module sui::validator_set {
         adjusted_storage_fund_reward_amounts: &vector<u64>,
         staking_rewards: &mut Balance<SUI>,
         storage_fund_reward: &mut Balance<SUI>,
+        new_epoch: u64,
```

Comment:
Guessing this can be derived from `ctx` (but don't see any changes at the call site of `distribute_reward`, so I'm not sure).